content/en/blog/_posts/2023-08-15-pkgs-k8s-io-introduction.md

```diff
@@ -173,7 +193,7 @@ publishing packages to the Google-hosted repository in the future.
    curl -fsSL https://pkgs.k8s.io/core:/stable:/v1.28/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
    ```
 
-_Update: In releases older than Debian 12 and Ubuntu 22.04, the folder `/etc/apt/keyrings` does not exist by default, and it should be created before the curl command._
+    _Update: In releases older than Debian 12 and Ubuntu 22.04, the folder `/etc/apt/keyrings` does not exist by default, and it should be created before the curl command._
```

Comment:
```suggestion
   _Update: In releases older than Debian 12 and Ubuntu 22.04, the folder `/etc/apt/keyrings` does not exist by default, and it should be created before the curl command._
```

Reply:
It's unfortunately not enough, it works only with 4 spaces
<img width="984" alt="image" src="https://github.com/kubernetes/website/assets/18719127/e3fb10b4-5537-40ef-9aab-6f5ebb0002bc">

Reply:
OK